components/CraftsList/CraftsList.tsx

```diff
@@ -322,14 +325,16 @@ export function CraftsList(props: Props) {
                 {!isLoggedIn || !accountInfo?.mcId ? <hr /> : ''}
             </div>
             <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-                <Form.Control style={{ width: selectWidth }} placeholder="Item name..." onChange={onNameFilterChange} />
-                <Form.Control style={{ width: selectWidth }} defaultValue={orderBy.value} as="select" onChange={updateOrderBy}>
+                <Form.Control style={{ width: '100%', marginRight: '3%' }} placeholder="Item name..." onChange={onNameFilterChange} />
+                <Form.Control style={{ width: '100%', marginRight: '3%' }} defaultValue={orderBy.value} as="select" onChange={updateOrderBy}>
                     {SORT_OPTIONS.map(option => (
                         <option value={option.value}>{option.label}</option>
                     ))}
                 </Form.Control>
+                <Form.Control style={{ width: '100%', marginRight: '3%' }} placeholder='Minimum Profit' onChange={onMinimumProfitChange} />
+
                 {profiles ? (
-                    <Form.Control style={{ width: selectWidth }} defaultValue={selectedProfile?.cuteName} as="select" onChange={onProfileChange}>
+                    <Form.Control style={{ width: '100%', marginLeft: '3%' }} defaultValue={selectedProfile?.cuteName} as="select" onChange={onProfileChange}>
```

Comment:
Is the wider space on the left of the profile box on purpose?
Also: Please move the css into the .css file and reuse it
![grafik](https://user-images.githubusercontent.com/58751503/218299782-dd6bed06-c8c1-40b2-9700-d26dc6ad1699.png)

Reply:
Oh, no, I'll fix it